src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -496,9 +497,9 @@ def _search_role_assignments(cli_ctx, assignments_client, definitions_client,
 
     if assignments:
         assignments = [a for a in assignments if (
-            not scope or
-            include_inherited and re.match(_get_role_property(a, 'scope'), scope, re.I) or
-            _get_role_property(a, 'scope').lower() == scope.lower()
+                not scope or
+                include_inherited and re.match(_get_role_property(a, 'scope'), scope, re.I) or
+                _get_role_property(a, 'scope').lower() == scope.lower()
```

Comment:
I would suggest you back out the formatting change and only submit the command change

Reply:
Fixed formatting - accidental change